problems.rb

```diff
@@ -0,0 +1,9 @@
+# TODO: Fix this whole function
+def printHi3TimesNTimes(n)
+  for i in 0..n
```

Comment:
Consider combining the two nested loops into one loop for code clarity

Reply:
Done!